godel/config/test-plugin.yml

```diff
@@ -2,3 +2,6 @@ tags:
   integration:
     names:
     - ^integration$
+exclude:
+  names:
+    - ^example$
```

Comment:
Without this,
```
$ ./godelw test
Error: [go list ./appconfig ./example ./githubapp ./oauth2] failed: main module (github.com/palantir/go-githubapp) does not contain package github.com/palantir/go-githubapp/example
: exit status 1
```

Should we just skip tests on example? Or is there a more correct way to enable tests and not have godel explode?